### DISCUSSION TITLE
pumgen format conversion error

### DISCUSSION THREAD

**Discussion by buzhiqiuyuebai** [SeisSol/SeisSol#797] [Q&A] [answered]
For the same grid file, gmsh and pumgen works fine when I set cl_1 = 5 or cl_1=1,And when I set cl_1 = 0.8 or cl_1=0.6, There is a problem（[pumgenError](https://github.com/buzhiqiuyuebai/A319/blob/main/pumgenError)） with format conversion in pumgen, do you know what caused this? The example I use is [here](https://github.com/buzhiqiuyuebai/A319/tree/main). Thank you in advance !

![image](https://user-images.githubusercontent.com/114474894/218703574-9d6217fe-3f57-4990-a45a-da9802ed7783.png)

**Comment by sebwolf-de** [comment, marked as answer]
Probably, your computer has not enough memory for a mesh with 169 million elements.

**Reply by buzhiqiuyuebai** [reply]
> Probably, your computer has not enough memory for a mesh with 169 million elements.

I use mpirun -n 8 pumgen loh.msh -s msh2, it can work.
